bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceHelper.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.voice.internal;
+
+import static java.util.Comparator.comparing;
+
+import java.util.Comparator;
+import java.util.Locale;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.voice.TTSService;
+import org.eclipse.smarthome.core.voice.Voice;
+
+/**
+ * {@link VoiceHelper} provides a utility method to consume sorted voices.
+ *
+ * @author Wouter Born - Initial contribution
+ */
+@NonNullByDefault
+public class VoiceHelper {
```

Comment:
Since `VoiceHelper` is only used in DS services I suggest to have this helper also be a DS component. By this we will also get rid of the `public static` method which is hard to mock/stub in tests.